src/NUnitFramework/tests/TextOutputTests.cs

```diff
@@ -139,14 +139,18 @@ public void TestContextOut_WritesToResult()
         [Test]
         public void TestContextWrite_WritesToResult()
         {
+#pragma warning disable NUnit1033 // The Write methods on TestContext will be marked as Obsolete and eventually removed
```

Comment:
disabled as the name of the test explicitly checks the deprecated method. `TestContextOut_WritesToResult()` above is tested separately